application/models/testutils.go

```diff
@@ -211,12 +211,23 @@ func createClaimFixture(tx *pop.Connection, policy Policy, config FixturesConfig
 }
 
 // CreateCategoryFixtures generates any number of category records for testing
+//   even indexed categories are Stationary and odd indexed ones are Mobile
 func CreateCategoryFixtures(tx *pop.Connection, n int) Fixtures {
 	CreateRiskCategories(tx)
 
 	categories := make(ItemCategories, n)
+	even := true
 	for i := range categories {
-		categories[i].RiskCategoryID = RiskCategoryMobileID()
+		if even {
```

Comment:
```suggestion
		if i % 2 == 0 {
```
I'm used to seeing this, but if you prefer the `even` variable, that's fine with me.

Reply:
:+1:

Not that it would have mattered for the test fixture creation, but I was thinking the even-odd thing would be faster than `i%2==0`.
But, when I run a timed test for the two different methods (for a million iterations),  `i%2==0` is a little faster.